microsoft/testsuites/vm_extensions/mdm.py

```diff
@@ -0,0 +1,97 @@
+from pathlib import PurePath
+from typing import Any, Dict, List, cast
+
+from assertpy import assert_that
+
+from lisa import Logger, Node, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.operating_system import CBLMariner, Debian, Posix, Ubuntu
+from lisa.tools import Curl, Echo, Service
+from lisa.util import LisaException, SkippedException, UnsupportedDistroException
+
+
+@TestSuiteMetadata(
+    area="MetricsExtension",
+    category="functional",
+    description="""
+    This test is a BVT for MDM MetricsExtension
+    """,
+)
+class MetricsExtension(TestSuite):
+    @TestCaseMetadata(
+        description="""
+            Verify whether MetricsExtension is installed, running,
+            and uninstalled successfully
+        """,
+        priority=1,
```

Comment:
Limit this test case for Azure, Ready only.

Reply:
Done